### PR TITLE
Use createUnsignedAttestation when creating attestations in ValidatorCoordinator

### PR DESCRIPTION
**Pull request by ajsutton**
## PR Description
Switch `ValidatorCoordinator` over to using `ValidatorApiChannel` to create unsigned attestations.

This is a "quick and dirty" switch over so it's still using `BeaconState` as part of the signing process which shouldn't be available to the validator client and it's still untested.  Will rebuild it properly over in the separate validator client with tests in a follow up PR. This just ensures that our REST API and validator are actually generating attestations the same way.

### CODE REVIEW

File: logging/src/main/java/tech/pegasys/teku/logging/StatusLogger.java [hunk 1]
```diff
@@ -72,7 +72,7 @@ public void blockCreationFailure(final Exception cause) {
     log.error("Error during block creation", cause);
   }
 
-  public void attestationFailure(final IllegalArgumentException cause) {
+  public void attestationFailure(final Throwable cause) {
```
Comment (cemozerr) [inline logging/src/main/java/tech/pegasys/teku/logging/StatusLogger.java:75]: Might be a good idea to name this as dutyFailure or something.

Comment (ajsutton) [reply logging/src/main/java/tech/pegasys/teku/logging/StatusLogger.java:75]: That message is specifically for failing to create an attestation - there's a separate status event for failing to create a block.

File: validator/coordinator/src/main/java/tech/pegasys/artemis/validator/coordinator/ValidatorCoordinator.java [hunk 2]
```diff
@@ -197,27 +194,82 @@ public void onAttestationEvent(BroadcastAttestationEvent event) throws IllegalAr
...
-      asyncProduceAttestations(
-          attesterInformations, headState, getGenericAttestationData(headState, headBlock));
-
-      // Save headState to check for slashings
-      //      this.headState = headState;
+      attestationAggregator.updateAggregatorInformations(attestersInformation);
+
+      final Map<UnsignedLong, List<AttesterInformation>> attestersByCommittee =
+          attestersInformation.stream()
+              .collect(Collectors.groupingBy(info -> info.getCommittee().getIndex()));
```
Comment (cemozerr) [inline validator/coordinator/src/main/java/tech/pegasys/artemis/validator/coordinator/ValidatorCoordinator.java:210]: Nice.

**Review by cemozerr — APPROVED** [review]
LGTM.

File: logging/src/main/java/tech/pegasys/teku/logging/StatusLogger.java (hunk 1 above)
Comment (cemozerr) [reply logging/src/main/java/tech/pegasys/teku/logging/StatusLogger.java:75]: Then we should probably change the log.warn() string below at some point.

**Pull request merged**